build-scripts/gulp/translations.js

```diff
@@ -61,14 +61,25 @@ function flatten(data) {
   return recursiveFlatten("", data);
 }
 
-function emptyFilter(data) {
+function hasHtml(data) {
+  return /<[a-z][\s\S]*>/i.test(data);
+}
+
+// Filter empty key's and key's that contain html
+function filterData(data, lang) {
```

Comment:
I don't want to filter it, I want the process to blow up and indicate which keys are bad, so that we can fix it.